test/libevmcore/CMakeLists.txt

```diff
@@ -1,5 +0,0 @@
-cmake_policy(SET CMP0015 NEW)
-
-aux_source_directory(. SRCS)
-
-add_sources(${SRCS})
```

Comment:
Why do you remove this?
The test in Assembly.cpp will not be included in testeth after this. Also every directory should have a CMake file.